### PR TITLE
DDCNL-4588-1: Fixed issue with being able to access 2021 from different pages

### PR DESCRIPTION
**Pull request by Christian-QA**

### CODE REVIEW

File: app/controllers/paye/PayeGovernmentSpendController.scala [hunk 1]
```diff
@@ -45,10 +45,13 @@ class PayeGovernmentSpendController @Inject()(
 
   def show(taxYear: Int): Action[AnyContent] = payeAuthAction.async { implicit request: PayeAuthenticatedRequest[_] =>
     payeAtsService.getPayeATSData(request.nino, taxYear).map {
+      case Right(_: PayeAtsData)
+          if (taxYear > appConfig.taxYear || taxYear < appConfig.taxYear - appConfig.maxTaxYearsTobeDisplayed) =>
+        Redirect(controllers.routes.ErrorController.authorisedNoAts(taxYear))
```
Comment (pascal22p) [inline app/controllers/paye/PayeGovernmentSpendController.scala:50]: That is not the response you used on the other pages? was is it not a forbidden response?

File: app/controllers/paye/PayeYourIncomeAndTaxesController.scala [hunk 2]
```diff
@@ -47,6 +47,10 @@ class PayeYourIncomeAndTaxesController @Inject()(
   def show(taxYear: Int): Action[AnyContent] = payeAuthAction.async { implicit request: PayeAuthenticatedRequest[_] =>
     {
       payeAtsService.getPayeATSData(request.nino, taxYear).map {
+        case Right(_: PayeAtsData)
+            if (taxYear > appConfig.taxYear || taxYear < appConfig.taxYear - appConfig.maxTaxYearsTobeDisplayed) => {
+          Redirect(controllers.routes.ErrorController.authorisedNoAts(taxYear))
```
Comment (pascal22p) [inline app/controllers/paye/PayeYourIncomeAndTaxesController.scala:52]: same here

**Review by pascal22p — CHANGES_REQUESTED** [review]

File: app/controllers/paye/PayeGovernmentSpendController.scala (hunk 1 above)
Comment (Christian-QA) [reply app/controllers/paye/PayeGovernmentSpendController.scala]: Done

File: app/controllers/paye/PayeYourIncomeAndTaxesController.scala (hunk 2 above)
Comment (Christian-QA) [reply app/controllers/paye/PayeYourIncomeAndTaxesController.scala]: Done

**Review by pascal22p — APPROVED** [review]

**Pull request merged**